pkg/goldpinger/k8s.go

```diff
@@ -16,14 +16,23 @@ package goldpinger
 
 import (
 	"log"
-
+	"io/ioutil"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func GetAllPods() map[string]string {
+func getNamespace() string {
+	b, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		log.Println("Unable to determine namespace: ", err.Error())
```

Comment:
Just to be explicit here, we should return empty string in this block.
```suggestion
 		log.Println("Unable to determine namespace: ", err.Error())
 		return ""
```